nailgun/entities.py

```diff
@@ -3114,6 +3114,8 @@ def poll(self, poll_rate=None, timeout=None):
             ``nailgun.entity_mixins.TASK_POLL_RATE``.
         :param timeout: Maximum number of seconds to wait until timing out.
             Defaults to ``nailgun.entity_mixins.TASK_TIMEOUT``.
+        :param must_succeed: Don't rise error when task finishes with other then success
+            result. Needed for negative tests
```

Comment:
As it is only required for negative tests, shouldn't this arg be set to False by default?

Reply:
Bad wording once again. True is good default, ability to set this to False is needed for negative tests, updated